### PR TITLE
Terms of Use update provisioning route

### PR DESCRIPTION
**Pull request by rileyanderson**
## Summary
- Create `PUT /v0/terms_of_use_agreements/update_provisioning` route that will update provisioing in MAP and set a cookie for CERNER

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#68934
- https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2406
## Testing 
- Sign-in using `oauth=false`
- PUT `v0/terms_of_use_agreements/update_provisioning` using postman interceptor, curl, fetch, or whatever
- you should get a 200 response with `{' provisioned': true }` and the `CERNER_CONSENT` cookie in the response headers and/or set in the browser. 

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
Terms Of Use

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected

### CODE REVIEW

File: app/services/terms_of_use/constants.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module TermsOfUse
+  module Constants
+    PROVISIONER_COOKIE_NAME = 'CERNER_CONSENT'
+    PROVISIONER_COOKIE_VALUE = 'ACCEPTED'
+    PROVISIONER_COOKIE_PATH = '/'
+    PROVISIONER_COOKIE_EXPIRATION = 2.minutes
```
Comment (asg5704) [inline app/services/terms_of_use/constants.rb:8]: Just curious.. should the cookie ever expire? 👀

Comment (joeniquette) [reply app/services/terms_of_use/constants.rb:8]: yea, otherwise if someone declines on another page then sites wont know

Comment (asg5704) [reply app/services/terms_of_use/constants.rb:8]: Okay.

**Review by ericboehs — APPROVED** [review]

**Review by asg5704 — APPROVED** [review]
✅  Approved

**Pull request merged**